lib/main.js

```diff
@@ -459,23 +473,23 @@ proto.hooks = {
       });
     });
 
+    var lastRating;
     var ratingObserver = new MutationObserver(function (mutations) {
-      mutations.forEach(function (m) {
+      var ratingsTouched = mutations.some(function (m) {
+        // Determine if our ratings were touched
         var target = m.target;
-        // If we are looking at a rating button and it's selected, emit a notification
-        // DEV: Prevent selection of container and "remove-circle-outline" button
-        // jscs:disable maximumLineLength
-        // Good:
-        //   <paper-icon-button icon="sj:thumb-up-outline" data-rating="5" role="button" tabindex="0" aria-disabled="false" class="x-scope paper-icon-button-0" title="Thumb-up" aria-label="Thumb-up"></paper-icon-button>
-        // Bad:
-        //   <div id="playerSongInfo" style=""></div>
-        //   <paper-icon-button icon="remove-circle-outline" data-rating="0" role="button" tabindex="0" aria-disabled="false" class="x-scope paper-icon-button-0"></paper-icon-button>
-        // jscs:enable maximumLineLength
```

Comment:
I feel like these comments would be good to preserve. Can we keep them?

It will be practical when Google changes their HTML on us and we have no idea what our original selectors were based on.